client/my-sites/marketplace/components/billing-interval-switcher/index.tsx

```diff
@@ -57,7 +89,25 @@ const BillingIntervalSwitcher: React.FunctionComponent< Props > = ( {
 	return (
 		<Container>
 			<PlansIntervalToggleLabel>{ translate( 'Price' ) }</PlansIntervalToggleLabel>
-			<PlansIntervalToggle intervalType={ billingPeriod } onChange={ onChange } />
+			<PlansIntervalToggle intervalType={ billingPeriod } onChange={ onChange }>
+				{ plugin ? (
+					<PluginAnnualSaving plugin={ plugin }>
+						{ ( annualSaving: { saving: any } ) =>
+							annualSaving.saving ? (
+								<>
+									<PluginAnnualSavingLabelDesktop>
+										(
+										{ sprintf( 'Save %(save)s', {
+											save: annualSaving.saving,
+										} ) }
+										)
```

Comment:
You can inject variables directly with the `translate` method:

```suggestion
										&nbsp;
										{ translate( 'Save %(save)s', {
											comment: 'Sale price label, ex: Save $51',
											args: { save: annualSaving.saving },
										} ) }
```

And just as a heads up: Usually you will actually want to use `sprintf` + `__` from `useI18n` like in [this example](https://github.com/Automattic/wp-calypso/blob/e862169d398950116a43d2f3528164008e6af091/client/my-sites/woocommerce/landing-page.tsx#L61-L67). We're slowly migrating away from the `translate` method to core wp functions but sticking with the legacy usage that's already in use here is totally fine in this sort of situation.